service/routes.py

```diff
@@ -130,7 +130,21 @@ def create_customers():
         jsonify(message), status.HTTP_201_CREATED, {"Location": location_url}
     )
 
+######################################################################
+# DELETE A PET
```

Comment:
@ddaisudov @stephrosenthal We probably want to change "DELETE A PET" to "DELETE A CUSTOMER"

Reply:
Done!